open_academy/models/partner.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
+
+
+class Partner(models.Model):
```

Comment:
As a good practice we use the same name that the parent model.

```suggestion
class ResPartner(models.Model):
```